src/card/Card.tsx

```diff
@@ -173,7 +203,7 @@ const NeoCard = ({
 const mapStateToProps = (state, ownProps) => ({
     report: getReportState(state, ownProps.index),
     editable: getDashboardIsEditable(state),
-    database: getDatabase(state),
+    database: getDatabase(state, ownProps.index, ownProps.dashboardSettings.pagenumber),
```

Comment:
it would make sense to change the order of the args here:
1. state
2. pagenumber
3. report number

(This would also be consistent with the action)

Reply:
Just pushed the update, now it has first the pageNumber and then the cardIndex